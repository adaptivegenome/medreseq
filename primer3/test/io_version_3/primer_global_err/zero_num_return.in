PRIMER_NUM_RETURN=0
MARKER_NAME=14.RHAP34FF2.seq
SEQUENCE=NNNNNNNNNNNNNNNNNNNNCCATTTGGAAAGTCAACCTTCAAGGCGTGTGTTCTTCCTCCTTGGCACCTTGAGGAGGCAGCAATGACATTTTGACAAATGCCTCAAGTTAGGTAGGTGCTCTATAGCAAAGAGCCAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNCGCGTGTACGAATGTGTGTAATGAAAGGGGAATATTCTGGCTTAGCTTACACATTAAGTTCATTCAAGGTCATTCAATAGTGGNNNNNNNNNNNNNNNNNNNNNNNNNNNN
INCLUDED_REGION=20,248
TARGET=141,44,TG
CL=GGAGGCAGCAATGACATTTT
CR=TACACACATTCGTACACGCG
=
MARKER_NAME=15.RHAP34FF3.seq
SEQUENCE=NNNNNNNNCACTGCCTGTCTCTGCTCTTTACTGAGAACAGTCAAGGGGACAGTATTGTTATCTCCACATATTTCACACTTTTCTTAACCCTTTCCTTAAGATTTTATCTCCTCCATTTTNNNNNNNNNNNNNNNNNNNNNNNNGCGCGCGGCCCNNNNNNNNNNNNNNNNNNNNNNNNNCTTTTGCTGGGCTCTTAAGTGGNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNN
INCLUDED_REGION=8,193
TARGET=119,24,AC
TARGET=154,25,CA
CL=TGAGAACAGTCAAGGGGACA
CR=CCACTTAAGAGCCCAGCAAA
=
