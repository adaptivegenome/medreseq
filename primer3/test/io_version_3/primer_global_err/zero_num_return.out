PRIMER_NUM_RETURN=0
MARKER_NAME=14.RHAP34FF2.seq
SEQUENCE=NNNNNNNNNNNNNNNNNNNNCCATTTGGAAAGTCAACCTTCAAGGCGTGTGTTCTTCCTCCTTGGCACCTTGAGGAGGCAGCAATGACATTTTGACAAATGCCTCAAGTTAGGTAGGTGCTCTATAGCAAAGAGCCAATACNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNCGCGTGTACGAATGTGTGTAATGAAAGGGGAATATTCTGGCTTAGCTTACACATTAAGTTCATTCAAGGTCATTCAATAGTGGNNNNNNNNNNNNNNNNNNNNNNNNNNNN
INCLUDED_REGION=20,248
TARGET=141,44,TG
CL=GGAGGCAGCAATGACATTTT
CR=TACACACATTCGTACACGCG
PRIMER_ERROR=PRIMER_NUM_RETURN < 1
=
